### PR TITLE
Fpet 932 email send reply

### PR DESCRIPTION
**Pull request by priyankahmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPET-932

### Change description ###

Notification emails for solicitors and citizens

Code changes done to implement send email to unrepresented party(applicant, respondent or other people)/ external parties.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by sairamkantheti1 — COMMENTED** [review]
Update jira ticket information and description for the pull request properly

File: src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java [hunk 1]
```diff
@@ -49,6 +49,6 @@ public enum EmailTemplateNames {
     SEND_AND_REPLY_NOTIFICATION_OTHER,
     TRANSFER_COURT_EMAIL_NOTIFICATION,
     LIST_ON_NOTICE_EMAIL_NOTIFICATION,
-    EMAIL_TO_LEGAL_REP_JUDGE_REJECTED_ORDER
+    EMAIL_TO_LEGAL_REP_JUDGE_REJECTED_ORDER,
```
Comment (sairamkantheti1) [inline src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java:52]: remove , gere

File: src/test/java/uk/gov/hmcts/reform/prl/services/SendAndReplyServiceTest.java [hunk 2]
```diff
@@ -1217,4 +1309,228 @@ public void testSendNotificationEmailOther() {
...
+            .sendOrReplyMessage(
+                SendOrReplyMessage.builder()
+                    .sendMessageObject(Message.builder()
+                                           .internalOrExternalMessage(InternalExternalMessageEnum.INTERNAL)
+                                           .externalMessageWhoToSendTo(null)
+                                           .messageAbout(MessageAboutEnum.APPLICATION)
+                                           .messageContent("some msg content")
+                                           .build())
+                    .respondToMessage(YesOrNo.No)
+                    .messages(messages)
```
Comment (sairamkantheti1) [inline src/test/java/uk/gov/hmcts/reform/prl/services/SendAndReplyServiceTest.java:1529]: remove src/main/java/uk/gov/hmcts/reform/prl/services/SendAndReplyService.java from suppression list and check the coverage of the changes

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -189,6 +190,7 @@ send-grid:
           SOA_PERSONAL_CA_DA_APPLICANT_LEGAL_REP: ${SEND_GRID_SOA_PERSONAL_CA_DA_APPLICANT_LEGAL_REP:d-13b61732cb744f89a9489d4b3500123a}
           SOA_SERVE_APPLICANT_SOLICITOR_NONPER_PER_CA_CB: ${SOA_SERVE_APPLICANT_SOLICITOR_NONPER_PER_CA_CB:d-8b6f127addd240b599528c75b3b37974}
           SOA_CA_LOCAL_AUTHORITY: ${SEND_GRID_SOA_CA_LOCAL_AUTHORITY:d-d121b680310f45e5a4117dd7023482e2}
+          SEND_EMAIL_TO_EXTERNAL_PARTY: d-d6f807d095b046458517924e6c48e5ea
   rpa:
```
Comment (sairamkantheti1) [inline src/main/resources/application.yaml:194]: add overriding configuration constant to include template for production

Comment (priyankahmcts) [reply src/main/resources/application.yaml:194]: Addressed

File: src/test/java/uk/gov/hmcts/reform/prl/services/SendAndReplyServiceTest.java (hunk 2 above)
Comment (priyankahmcts) [reply src/test/java/uk/gov/hmcts/reform/prl/services/SendAndReplyServiceTest.java:1529]: Addressed

File: src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java (hunk 1 above)
Comment (priyankahmcts) [reply src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java:52]: Removed

**Pull request merged**